cxx-qt/README.md

```diff
@@ -6,10 +6,10 @@ SPDX-FileContributor: Gerhard de Clercq <gerhard.declercq@kdab.com>
 SPDX-License-Identifier: MIT OR Apache-2.0
 -->
 
-# cxx-qt
+# CXX-Qt
```

Comment:
We should probably go through the READMEs and ensure that they have up to date info and describe the crates well.